packages/common/src/index.ts

```diff
@@ -320,3 +320,25 @@ export const convertStatusToText = (
       return "PENDING";
   }
 };
+
+export const prefixZero = (i: number): string =>
+  i < 10 ? "0" + i : i.toString();
+
+export const getUTCDate = (date: Date): string => {
+  const utcDate = [
+    prefixZero(date.getUTCDate()),
+    prefixZero(date.getUTCMonth() + 1),
+    prefixZero(date.getUTCFullYear()),
+  ];
+
+  return utcDate.join("/");
+};
```

Comment:
Naming: `formatUTCDateAsISOString`

Also, `toISOString` should return the date in UTC:

```
export const formatUTCDateAsISOString = (date: Date): string => {
  const isoString = date.toISOString();
  return isoString.slice(0, 10).replace(/-/g, "/");
};
```

Reply:
good idea - done.